samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -1332,7 +1528,11 @@ static void radio_prepare(bool rx)
 	 * between READY event and START task and
 	 * between END event and DISABLE task
 	 */
-#if DIRECTION_FINDING_SUPPORTED
+#if CONFIG_SOC_SERIES_NRF54LX
+	nrf_radio_shorts_set(NRF_RADIO,
+			     NRF_RADIO_SHORT_READY_START_MASK |
+			     NRF_RADIO_SHORT_PHYEND_DISABLE_MASK);
```

Comment:
Maybe this phyend_disable/end_disable could be defined and used here without ifdef for soc?